apps/web/src/pages/SearchRequests/RequestPage/RequestPage.tsx

```diff
@@ -36,6 +36,7 @@ const RequestPage = () => {
           id: "9Jkoms",
           description: "Title displayed on hero for Search and Request pages.",
         })}
+        subtitle="Submit your filtered request for talent."
```

Comment:
This needs to be wrapped in `intl.formatMessage`, otherwise it will not be able to be translated.

Reply:
Updated in [localized strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8485/commits/8978e0a26fbdfb8875ddaea4bf51bb96da639515)